client/js/views/visualize.react.jsx

```diff
@@ -152,7 +151,7 @@ var Visualize = React.createClass({
       //<File key={this.state.currentPath + '/' + this.state.filePaths[this.state.currentPath].commitIndex} currentIndex={this.state.commitIndex} filePaths={this.state.filePaths} currentPath={this.state.currentPath}/>
       return (
         <Col xs={9} md={9} style={{height: this.state.windowHeight, overflow: 'scroll'}}>
-          <File currentIndex={this.state.commitIndex} urls={{to : '', from: ''}} filePaths={this.state.filePaths} currentPath={this.state.currentPath}/>
+          <File urls={{to : '', from: ''}} filePaths={this.state.filePaths} currentPath={this.state.currentPath}/>
```

Comment:
refactor out file urls prop for normal file view (just checking that it is a prop for diffualize file view). diffualize file can have urls as its own props